compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenInterfaces.td

```diff
@@ -9,6 +9,8 @@
 
 include "iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenDialect.td"
 
+def TensorTypeAttr : TypeAttrBase<"TensorType", "Tensor type attribute">;
+
```

Comment:
What is this change for?

Reply:
Small refactoring. There is already a `TensorTypeAttr` so I converged both into the interface td file.